nilearn/regions/parcellations.py

```diff
@@ -116,6 +179,59 @@ def _labels_masker_extraction(img, masker, confound):
     return signals
 
 
+def _apply_surf_mask_on_labels(mask_data, labels_data, background_label=0):
+    """Apply mask to labels data.
+    Copy of nilearn.maskers.surface_labels_masker._apply_mask.
```

Comment:
may be good to extract this somewhere else instead of copying it, no?

Reply:
I agree! Similar functions for volumes are in nilearn.regions.signal_extraction. So maybe that's the best place

Reply:
I would say to maybe keep it as function in nilearn.maskers.surface_labels_masker and then think later if we can find it a proper home